ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -468,12 +484,12 @@ std::tuple<ttnn::Shape, ttnn::MemoryConfig, bool> get_conv_padded_input_shape_an
             1,
             input_tensor_height_snapped_to_tile,
             input_tensor_width_snapped_to_channels_alignment});  // TODO: resolve ttnn::types::Shape and
-                                                                 // tt::tt_metal::LegacyShape issue to clean up next line
+        uint32_t tile_size = !conv_config.use_non_tile_height ? 32 : 1;
```

Comment:
how can tile_size be 1? Pls use TILE_HEIGHT or TILE_WIDTH instead of 32

Reply:
Sure I will rename the variable.